schema/deploy/shipping/views.sql

```diff
@@ -1207,7 +1211,7 @@ create or replace view shipping.hcov19_observation_v1 as
         ) as deduplicated_hcov19_bbi
         left join jsonb_to_recordset(replicates) as r("crt" text) on true
 
-        group by sample_id, hcov19_result_received_bbi, hcov19_present_bbi
+        group by sample_id, hcov19_result_received_bbi, hcov19_present_bbi, hcov19_device_bbi, hcov19_assay_type_bbi
```

Comment:
It would be good to look at the distinct'ing and group'ing in here in the future to see if this is the right way to come up with one row per sample. Since those operations are not being added via this change, that doesn't hold up this commit.